firefox_ui_tests/remote/security/test_ssl_disabled_error_page.py

```diff
@@ -0,0 +1,55 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+
+from marionette import By
+from marionette.errors import MarionetteException
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestSSLDisabledErrorPage(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.url = 'https://tlsv1-0.mozqa.com'
+
+        tabbar = self.browser.tabbar
+        tabbar.close_all_tabs([tabbar.tabs[0]])
+
+        # Disable SSL 3.0, TLS 1.0 and TLS 1.1 for secure connections
+        # by forcing the use of TLS 1.2
+        # see: http://kb.mozillazine.org/Security.tls.version.*#Possible_values_and_their_effects
+        self.prefs.set_pref('security.tls.version.min', 3)
+        self.prefs.set_pref('security.tls.version.max', 3)
+
+        self.marionette.set_context('content')
+
+    def tearDown(self):
+        self.prefs.restore_all_prefs()
+        FirefoxTestCase.tearDown(self)
+
+    def test_ssl_disabled_error_page(self):
+        # Open the test page
+        self.assertRaises(MarionetteException, self.marionette.navigate, self.url)
+
+        # Wait for the DOM to receive events
+        time.sleep(1)
+
+        # Verify "Secure Connection Failed" error page title
+        title = self.marionette.find_element(By.ID, 'errorTitleText')
+
+        nss_failure2title = self.browser.get_entity('oldSecurityProtocol.title')
+        self.assertEquals(title.get_attribute('textContent'), nss_failure2title)
```

Comment:
This has been changed yesterday for Nightly. Please sync with the already updated Mozmill test on bug 1135616.